plugins/com.google.cloud.tools.eclipse.preferences/plugin.xml

```diff
@@ -14,18 +14,12 @@
       </page>
    </extension>
    <extension
```

Comment:
You mean including everything Analytics-related, such as `AnalyticsOptInArea.java` and `AnalyticsPreferences.java` in this bundle, right?